lib/Checkout/CheckoutApiException.php

```diff
@@ -20,7 +20,7 @@ public static function from(RequestException $requestException): CheckoutApiExce
     {
         $body = json_decode($requestException->getResponse()->getBody()->getContents(), true);
         $ex = new CheckoutApiException("The API response status code (" . $requestException->getCode() . ") does not indicate success.");
-        $ex->request_id = $body != null ? $body["request_id"] : null;
+        $ex->request_id = $body != null ? $body["request_id"] ?? null : null;
```

Comment:
This probably is not going to work.
What do you think
`$ex->request_id = $body != null && array_key_exists("request_id", $body) ? $body["request_id"] : null;`